5637 time based log rollover to staging (#5673)
OpenLiberty/docs: modules/ROOT/pages/access-logging.adoc

== HTTP access log settings

You can configure HTTP access log settings either for multiple endpoints that share common log settings or for individual endpoints.

Settings for a common log::

> To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file, and then reference it from multiple `httpEndpoint` elements, as shown in the following example:

Reviewer: ```suggestion
To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file. Reference this element from multiple `httpEndpoint` elements, as shown in the following example.
```

> To enable logging for multiple endpoints with common settings, include `httpAccessLogging` as a top-level element in your `server.xml` file, and then reference it from multiple `httpEndpoint` elements, as shown in the following example:

Reviewer: Re-phrased the sentence as Acrolinx picked up this for being too long. Also, removed : and replaced with a period.

[source,xml]
----
<httpAccessLogging id="accessLogging"/>
<httpEndpoint id="defaultHttpEndpoint" accessLoggingRef="accessLogging" httpPort="9080" httpsPort="9443"/>
<httpEndpoint id="otherHttpEndpoint" accessLoggingRef="accessLogging" httpPort="9081" httpsPort="9444"/>
----

Settings for distinct logs for each endpoint::

> To enable logging for individual endpoints, use an `accessLogging` child element and specify a file path that does not conflict with other logs, as shown in the following example:

Reviewer: ```suggestion
To enable logging for individual endpoints, use an `accessLogging` child element and specify a file path that does not conflict with other logs, as shown in the following example.
```

== HTTP access log format

> Use the `accessLogFormat` property to specify the information and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.

Reviewer: Acrolinx picked this up for being too long.

Author: I see these sentences as being 26 and 11 words long- not too long.

> This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples.

Reviewer: Acrolinx picked this up for being too long.

> This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples.

Author: ```suggestion
Specify the log format string with the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples.
```

> In the following `server.xml` file example, the `logformat` attribute for the `httpAcccessLogging` element specifies the log format string for multiple endpoints that reference the `accessLogging` configuration ID.

Author: ```suggestion
In the following `server.xml` file example, The `logformat` attribute for the `httpAcccessLogging` element specifies the log format string. Multiple endpoints can specify this string by referencing the `accessLogging` configuration ID.
```

> In the following `server.xml` file example, the `logformat` attribute for the `httpAcccessLogging` element specifies the log format string for multiple endpoints that reference the `accessLogging` configuration ID.

Reviewer: Acrolinx picked this up for being too long.
